trending_data/src/main/java/com/google/sps/data/YTVid.java

```diff
@@ -14,14 +14,17 @@
 
 package com.google.sps.data;
 
-/** item coding comment. */
+/** class for storing and representing Youtube Videos
+*   contains video's id and html link
+*   construct element of class with id 
+*/
 public final class YTVid {
 
   private final String id;
   private final String link;
```

Comment:
This `link` is unused. It's being set here in the java class, but nobody is reading it. Javascript is also doing this URL construction too (`createIframeById` takes only a video ID, and generates the link itself), so either this should be removed or javascript should use the .link field in this class.